eslint.config.mjs

```diff
@@ -0,0 +1,143 @@
+import { FlatCompat } from '@eslint/eslintrc';
+import js from '@eslint/js';
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import tsParser from '@typescript-eslint/parser';
+import react from 'eslint-plugin-react';
+import globals from 'globals';
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+  recommendedConfig: js.configs.recommended,
+  allConfig: js.configs.all,
+});
+
+export default [
+  {
+    ignores: [
+      '**/dist',
+      '**/e2e',
+      '**/node_modules',
+      '**/assets',
+      '**/coverage',
+      '**/build',
+      'packages/web/*',
+      'packages/elvis/checkDeprecations.js',
+      'packages/elvis/elvis.js',
+      'packages/elvis/elvis.full.js',
+      'packages/elvis/src/templates/*.js',
+      'packages/react-test/*',
+      'packages/vue-test/*',
+      'packages/components/playwright-report/*',
+      'packages/components/test-results/*',
+      'packages/components/__snapshots__/*',
+    ],
+  },
+  {
+    plugins: {
+      '@typescript-eslint': typescriptEslint,
+    },
+
+    languageOptions: {
+      globals: {
+        ...globals.node,
+      },
+
+      ecmaVersion: 'latest',
+      sourceType: 'module',
+
+      parserOptions: {
+        ecmaFeatures: {
+          jsx: true,
+          modules: true,
+        },
+      },
+    },
+
+    settings: {
+      react: {
+        version: 'detect',
+      },
+    },
+
+    rules: {
+      'no-console': [
+        'error',
+        {
+          allow: ['warn', 'clear', 'info', 'error', 'dir', 'trace', 'table'],
+        },
+      ],
+
+      'func-names': 'off',
+    },
+  },
+  ...compat
+    .extends(
+      'eslint:recommended',
+      'plugin:@typescript-eslint/recommended',
+      'plugin:@typescript-eslint/eslint-recommended',
+      // 'plugin:@typescript-eslint/stylistic',
```

Comment:
Is this plugin incompatible?

Reply:
It's recommended to be there (it's there when you generate a new config), however it caused too many errors for me to bother fixing it 🤷 It's stuff like ordering members of classes correctly (private variables vs public variables vs methods etc.)

Reply:
I could always just remove it completely instead of commenting it out

Reply:
Ah, gotcha